jangjuseong/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL = URL
+
+PORT = PORT_NUMBER
+
+TYPEORM_CONNECTION = CONNECTION
+TYPEORM_HOST = HOST
+TYPEORM_USERNAME = PASSWORD
+TYPEORM_PASSWORD = wntjdaos
```

해당 부분도 위 리뷰를 참고하여 수정해주면 좋을 것 같습니다.
```suggestion
TYPEORM_USERNAME = USERNAME
TYPEORM_PASSWORD = PASSWORD
```